sapling_jubjub.py

```diff
@@ -142,6 +142,14 @@ def __str__(self):
 JUBJUB_COFACTOR = Fr(8)
 
 class Point(object):
+    @staticmethod
+    def rand(rand):
+        while True:
+            data = rand.b(32)
+            p = Point.from_bytes(data)
+            if p:
+                return p
+
```

Comment:
For clarity, ``if not u:`` in the ``from_bytes`` method on line 167 should be ``if u is not None:``. (It's technically equivalent because a not-None ``Fq`` object is always truthy even if it represents zero. However, that took me about 30 seconds to think about.)

Reply:
I assume you mean that it should be `if u is None:`